tools/protocols/pull-proto.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Configuration parameters
+REPO_URL="https://github.com/apache/skywalking-data-collect-protocol.git"
```

Comment:
Why you don't use the submodule to import the data collect protocol dependency repo? It easy to control the version, rather than using git command.